src/Containers/CarouselTestimonial/CarouselTestimonial.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Carousel } from 'react-responsive-carousel';
+import SmallText from '@Text/SmallText';
+import styled from 'styled-components';
+import 'react-responsive-carousel/lib/styles/carousel.min.css';
+
+export interface ICarouselTestimonialProps {
+    carouselTitle: string;
+    carouselImage: string;
+    reviewDescriptions: string[];
+    reviewers: string[];
+    carouselAutoplay: boolean;
+    carouselInterval: number;
+    carouselLoop: boolean;
+}
+
+export const CarouselTestimonial: React.FC<ICarouselTestimonialProps> = ({
+    carouselTitle,
+    carouselImage,
+    reviewDescriptions,
+    reviewers,
+    carouselAutoplay,
+    carouselInterval,
+    carouselLoop,
+}) => {
+    /**
+     * Returns the JSX element of each carousel item based on its index
+     */
+    const getCarouselItem = (index: number) => (
+        <Section key={index}>
+            <TitleDiv key={carouselTitle}>{carouselTitle}</TitleDiv>
+            <ImageDiv key={carouselImage}>
+                <img src={carouselImage} />
+            </ImageDiv>
+            <ReviewTextDiv>
+                <SmallText color="#006aff" size="h4" bold>
```

Comment:
Use Theme Colors not Magic Coloors

Reply:
video:

https://user-images.githubusercontent.com/63363693/113187668-1e05ab80-9227-11eb-875d-8c449bf0e9bc.mp4